lib/ReactViews/Map/Navigation/ZoomControl.jsx

```diff
@@ -86,6 +86,19 @@ const ZoomControl = React.createClass({
             const direction = Cartesian3.subtract(focus, camera.position, cartesian3Scratch);
             const movementVector = Cartesian3.multiplyByScalar(direction, 2.0 / 3.0, cartesian3Scratch);
             const endPosition = Cartesian3.add(camera.position, movementVector, cartesian3Scratch);
+
+            // make sure that we don't crush the terrain through
+            const pickRay = new Ray(scene.camera.positionWC, scene.camera.directionWC);
+            const globe = scene.globe;
+            const surfacePosition = globe.pick(pickRay, scene);
```

Comment:
`globe.pick` will return undefined if the ray doesn't insersect the globe (i.e. if you tip the camera so that the globe horizon is below the center of the screen).  We need to make sure we don't crash in that scenario.